pkg/counter/cache.go

```diff
@@ -90,7 +98,7 @@ func DecrementMasterUserRecordCount(logger logr.Logger, domain metrics.Domain) {
 		} else {
 			logger.Error(fmt.Errorf("the count of MasterUserRecords is zero"), "unable to decrement the number of MasterUserRecords")
 		}
-		logger.Info("decrementing MasterUserRecordGauge", "value", cachedCounts.MasterUserRecordCount)
+		log.Info("decrementing MasterUserRecordGauge", "value", cachedCounts.MasterUserRecordCount)
```

Comment:
Why changing it to `log`?

Reply:
oh that's an mistake 😳
I'll revert that

Reply:
reverted in 41272d8